src/content/api/module-methods.md

```diff
@@ -163,7 +164,7 @@ T> Note that all options can be combined like so `/* webpackMode: "lazy-once", w
 
 T> Note that `webpackInclude` and `webpackExclude` options do not interfere with the prefix. eg: `./locale`.
 
-`webpackExports`: tells webpack to only bundle the used exports of a module when using dynamic imports. It can decrease the output size of a chunk. Available since [webpack 5.0.0-beta.18](https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.18).
+`webpackExports`: tells webpack to only bundle the specified exports of a module. It can decrease the output size of a chunk. Available since [webpack 5.0.0-beta.18](https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.18).
```

Comment:
```suggestion
`webpackExports`: tells webpack to only bundle the specified exports of a dynamically `import()`ed module. It can decrease the output size of a chunk. Available since [webpack 5.0.0-beta.18](https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.18).
```
How about changing to this?

Reply:
The whole section is about dynamic import, so I see this as redundant. But I won't insist. I think you have access to edit my branch, so you can apply the suggestion?

Reply:
People might copy that paragraph and use it somewhere without the context of `dynamic import` section, hence I prefer to keep it in this case.